execution/evm/vm.go

```diff
@@ -164,10 +165,6 @@ func (vm *VM) Call(callState *state.Cache, caller, callee *acm.MutableAccount, c
 			// Copy any state updates from child call frame into current call frame
 			childCallState.Sync(callState)
 		}
-		if vm.stackDepth == 0 {
```

Comment:
We'll probably need to talk over this because I'm still not entirely sure how else we would resolve nested errors in the test cases. Surely we don't want to swallow any errors?